crates/biome_css_formatter/tests/specs/css/variables.css.snap

```diff
@@ -91,8 +91,21 @@ Line width: 80
 	prop: var(--prop1) var(prop2);
 	prop: var(--prop1) var(prop2);
 
-	prop: hsl(var(--hue, 0) var(--sat, 100) var(--light, 1));
+	prop: hsl(
+			var(--hue, 
+        0) 
+        var(
+            --sat, 100
+            
```

Comment:
There is a change in formatting.
What do you think? @faultyserver

Reply:
There are a few of these, and all of them are expected because the syntax tree structure has changed. I'm happy to come do a follow up on this afterwards to re-implement these nodes. I don't want to add the burden of fixing formatting changes to anyone contributing to the parser since it's still in progress and will inevitably change frequently.